### PR TITLE
chg: usr: SDK-283 Added implementation for parameters for commands history api

### PR DESCRIPTION
**Pull request by meghapthakkar**

### CODE REVIEW

File: src/main/java/com/qubole/qds/sdk/java/details/CommandApiImpl.java [hunk 1]
```diff
@@ -147,4 +153,32 @@ public NotebookCommandBuilder notebook()
...
 
+    @Override
+    public CommandApi startDate(String startDate)
+    {
+        parameters.put("start_date", startDate);
+        return this;
+    }
+
+    @Override
+    public CommandApi allUsers(int allUsers)
```
Comment (amoghmargoor) [inline src/main/java/com/qubole/qds/sdk/java/details/CommandApiImpl.java]: either take this a s boolean or check for non 0 or 1

File: src/main/java/com/qubole/qds/sdk/java/details/CommandApiImpl.java [hunk 2]
```diff
@@ -147,4 +153,32 @@ public NotebookCommandBuilder notebook()
...
+    public CommandApi includeQueryProperties(boolean includeQueryProperties)
+    {
+        parameters.put("include_query_properties", Boolean.toString(includeQueryProperties));
+        return this;
+    }
+
+    @Override
+    public CommandApi endDate(String endDate)
+    {
+        parameters.put("end_date", String.valueOf(endDate));
```
Comment (amoghmargoor) [inline src/main/java/com/qubole/qds/sdk/java/details/CommandApiImpl.java]: endDate is already a string -> u need not have to again do String.valueOf()

**Comment by amoghmargoor** [comment]
Approved - apart from minor nit.

**Comment by meghapthakkar** [comment]
Made changes as per Amogh's comments

**Pull request merged**